### ISSUE TITLE
Dependencies do not comply with psr-4 autoloading

### ISSUE THREAD

**Issue by phil-davis** [icewind1991/Streams#13]
See https://github.com/owncloud/core/issues/37159

```
composer update
Loading composer repositories with package information
Updating dependencies (including require-dev)
Nothing to install or update
Package container-interop/container-interop is abandoned, you should avoid using it. Use psr/container instead.
Package guzzlehttp/ringphp is abandoned, you should avoid using it. No replacement was suggested.
Package guzzlehttp/streams is abandoned, you should avoid using it. No replacement was suggested.
Writing lock file
Generating optimized autoload files
Deprecation Notice: Class ZipStreamer\Count64Base located in ./lib/composer/deepdiver/zipstreamer/src/lib/Count64.php does not comply with psr-4 autoloading standard. It will not autoload anymore in Composer v2.0. in phar:///usr/local/bin/composer/src/Composer/Autoload/ClassMapGenerator.php:201
Deprecation Notice: Class ZipStreamer\Count64_32 located in ./lib/composer/deepdiver/zipstreamer/src/lib/Count64.php does not comply with psr-4 autoloading standard. It will not autoload anymore in Composer v2.0. in phar:///usr/local/bin/composer/src/Composer/Autoload/ClassMapGenerator.php:201
Deprecation Notice: Class ZipStreamer\Count64_64 located in ./lib/composer/deepdiver/zipstreamer/src/lib/Count64.php does not comply with psr-4 autoloading standard. It will not autoload anymore in Composer v2.0. in phar:///usr/local/bin/composer/src/Composer/Autoload/ClassMapGenerator.php:201
Deprecation Notice: Class ZipStreamer\Count64 located in ./lib/composer/deepdiver/zipstreamer/src/lib/Count64.php does not comply with psr-4 autoloading standard. It will not autoload anymore in Composer v2.0. in phar:///usr/local/bin/composer/src/Composer/Autoload/ClassMapGenerator.php:201
Deprecation Notice: Class Icewind\Streams\Tests\PartialWrapper located in ./lib/composer/icewind/streams/tests/RetryWrapper.php does not comply with psr-4 autoloading standard. It will not autoload anymore in Composer v2.0. in phar:///usr/local/bin/composer/src/Composer/Autoload/ClassMapGenerator.php:201
Deprecation Notice: Class Icewind\Streams\Tests\FailWrapper located in ./lib/composer/icewind/streams/tests/RetryWrapper.php does not comply with psr-4 autoloading standard. It will not autoload anymore in Composer v2.0. in phar:///usr/local/bin/composer/src/Composer/Autoload/ClassMapGenerator.php:201
Deprecation Notice: Class Icewind\Streams\Tests\RetryWrapperTest located in ./lib/composer/icewind/streams/tests/RetryWrapper.php does not comply with psr-4 autoloading standard. It will not autoload anymore in Composer v2.0. in phar:///usr/local/bin/composer/src/Composer/Autoload/ClassMapGenerator.php:201
Deprecation Notice: Class Icewind\Streams\UrlCallback located in ./lib/composer/icewind/streams/src/UrlCallBack.php does not comply with psr-4 autoloading standard. It will not autoload anymore in Composer v2.0. in phar:///usr/local/bin/composer/src/Composer/Autoload/ClassMapGenerator.php:201
```

**Comment by kesselb** [comment]
Should be fixed by https://github.com/icewind1991/Streams/commit/77d750ccc654c0eda4a41fedb2dbd71053755790 and https://github.com/icewind1991/Streams/commit/f1d138edddbfb5e5d894a5b4d980b98c3c48aa35. 

@phil-davis Mind to close the issue if the problem is solved?

**Comment by peter279k** [comment]
It seems that it's fixed on `v0.7.2` version.

And I think this issue should be closed.

**Comment by phil-davis** [comment]
"Deprecation Notice" is no longer emitted - fixed.
